AlgebraInLean/World01/Sheet2.lean

```diff
@@ -0,0 +1,52 @@
+import AlgebraInLean.World01.Sheet1
+namespace Sheet2
+
+/- Groups aren't the only important structures in algebra. By adding or
+subtracting certain properties from our definition, we create structures with
+different properties.-/
+
+/- The simplest structure in algebra is a magma, which only requires a closed operation-/
+class Magma (G : Type*) where
```

Comment:
At some point, you should elaborate on `Type*`. You don't have to give a whole schpiel on universes, but you should at least mention it.

Reply:
Yes, you can link them to the relevant documentation if it gets too long